articles/other/other-ref-idam.md

```diff
@@ -20,6 +20,15 @@ UKCloud's Identity & Access Management API (IDAM) enables you to manage user acc
 
 Full documentation for the IDAM API is available on [SwaggerHub](https://app.swaggerhub.com/apis/ukcloud/idam-service).
 
+## Platforms using IDAM
+
+IDAM is in use on the UKCloud for OpenStack plaforms, and for newer UKCloud for OpenShift deployments.
```

Comment:
Suggest changing to:

IDAM is used for UKCloud for OpenStack and for newer UKCloud for OpenShift clusters.

Reply:
Personally I'd prefer:

> IDAM is compatible with UKCloud for OpenStack and newer UKCloud for OpenShift clusters.